packages/react/src/app-layout/__snapshots__/AppLayout.test.tsx.snap

```diff
@@ -403,13 +403,11 @@ exports[`AppLayout renders correctly 1`] = `
                 width="24"
                 xmlns="http://www.w3.org/2000/svg"
               >
-                <circle
-                  cx="12"
-                  cy="12"
-                  r="10.5"
+                <path
```

Comment:
Changed all existing `circle` shapes to the `path` equivalent so they all render identically. Otherwise, `circle` was a bit thicker.